Sources/RxMoya/RxMoyaProvider.swift

```diff
@@ -6,22 +6,35 @@ import Moya
 
 /// Subclass of MoyaProvider that returns Observable instances when requests are made. Much better than using completion closures.
 open class RxMoyaProvider<Target>: MoyaProvider<Target> where Target: TargetType {
+    
+    /// Propagated to Alamofire as callback queue. If nil - main queue will be used.
+    fileprivate let queue: DispatchQueue?
+    
     /// Initializes a reactive provider.
-    override public init(endpointClosure: @escaping EndpointClosure = MoyaProvider.defaultEndpointMapping,
+    public init(endpointClosure: @escaping EndpointClosure = MoyaProvider.defaultEndpointMapping,
```

Comment:
Sorry, i'm afraid you didn't get me. You are right, old init calls will not break after edit(case queue param is optional), but any RxSwift/ReactiveSwift provider descendants overridden init will not compile, until user remove `override`. Let me show you what i'm talking about:
#### This is Moya in current state with two user created provider descendants
![This is Moya in current Moya state with two user created provider descendants](https://cloud.githubusercontent.com/assets/6107211/25172990/53224cee-24fb-11e7-8d94-a4a84d460ed9.png)

#### This is Moya state, after adding queue param(note that new param has default value)
![screen shot 2017-04-19 at 12 22 07 pm](https://cloud.githubusercontent.com/assets/6107211/25173091/95c98fc6-24fb-11e7-88fa-1fddbf5f7c4e.png)

#### And finally that is was i proposed in previous commit
![screen shot 2017-04-19 at 12 22 52 pm](https://cloud.githubusercontent.com/assets/6107211/25173106/a4e831ba-24fb-11e7-905d-b970e9ccd20b.png)

Adding queue param to `MoyaProvider` will do even more mess, if we will stick to replace instead of deprecation